JobsDeviOS/Infra/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.7
+
+import PackageDescription
+
+let package = Package(
+    name: "Infra",
```

Comment:
_nit_: Eu recomendaria definir a versão da plataforma que seu pacote dará suporte. Essa é uma boa prática para ajudar a manter, dar suporte a algumas funcionalidades específicas e a retrocompatibilidade do seu pacote quando você o estiver expondo pra algumas aplicações.
```suggestion
    name: "Infra",
    platforms: [
        .iOS(.v16)
    ]
```

Reply:
Bom ponto, tinha esquecido.